odl/space/ntuples.py

```diff
@@ -1320,7 +1349,7 @@ def adjoint(self):
                                       ''.format(self.domain.field,
                                                 self.range.field))
         return MatVecOperator(self.range, self.domain,
-                              np.asarray(np.asmatrix(self.matrix).H))
+                              self.matrix.conj().T)
```

Comment:
Given the fact that it can be either an `np.ndarray` or a `sp.sparse.spmatrix`, I don't really know how to call it other than `matrix`. That what it very much stands for, after all it's an operator of matrix multiplication with a vector. And even if somebody gives a `numpy.matrix`, it's internally converted to `np.ndarray`.